B-Sockets-ServerTesting/sockets-servertesting.md

```diff
@@ -0,0 +1,286 @@
+Sockets and Server Testing
+===========================
+Sockets and HTTP
+---------------------------
+### What is a socket?
```

Comment:
add blank line between headers pls

personally I prefer using `#` and `##` over `==` and `--` since it's more consistent across header levels, but this is nitpicky and not smt we're consistent about throughout the course materials